explorer/tests/test_exporters.py

```diff
@@ -75,4 +75,28 @@ def test_writing_excel(self):
 
         expected = b('PK')
 
-        self.assertEqual(res[:2], expected)
\ No newline at end of file
+        self.assertEqual(res[:2], expected)
+
+
+
+@unittest.skipIf(sys.version_info[0] > 2,  "not supported in this library version")
```

Comment:
I think the move is to probably add a utility function like

```
def pdf_supported():
    # try to import the dependency
    # and python version == 2
    return True else False
```

Then use that both here and in app_settings.py